src/test/java/com/iridium/testplugin/managers/TeamManager.java

```diff
@@ -91,7 +91,7 @@ public List<TestTeam> getTeams() {
 
     @Override
     public List<User> getTeamMembers(TestTeam team) {
-        return UserManager.users.stream().filter(user -> user.getActiveProfile().getTeamID() == team.getId()).collect(Collectors.toList());
+        return UserManager.users.stream().filter(user -> UserManager.profiles.stream().filter(profile->profile.getUuid().equals(user.getUuid())).anyMatch(profile->profile.getTeamID() == team.getId())).collect(Collectors.toList());
```

Comment:
since the functionality has changed, would be good to get some tests covering this. maybe in the members gui just a test or something to check this